### ISSUE TITLE
Library not loaded: @rpath/liblua54.framework/Versions/A/liblua54 on MAUI, Mac OS, Mac Catalyst

### ISSUE THREAD

**Issue by AndrewPlakhotnyi** [NLua/NLua#453]
Steps to reproduce:
1. Install MAUI development environment on Mac OS using [this guide ](https://github.com/dotnet/maui/wiki/macOS-Install)
2. Create MAUI project
3. Add NLua 1.6.0 Nuget
4. Build and run the project `dotnet build -f net6.0-maccatalyst t:Run`

The app crashes with the error `Library not loaded: @rpath/liblua54.framework/Versions/A/liblua54`. 

The actual liblua54 library is located at `@rpath/liblua54.framework/liblua54`, so the following workaround helps: 
`install_name_tool -change @rpath/liblua54.framework/Versions/A/liblua54 @rpath/liblua54.framework/liblua54 SingleMauiApp`

However, this workaround breaks the code signature of the file.

Basically, liblua54 should not be linked to @rpath/liblua54.framework/Versions/A/liblua54, the path is wrong and should be  `@rpath/liblua54.framework/liblua54` Do you have any idea

**Comment by viniciusjarina** [comment]
I guess this is probably an error on .NET ios workload. KeraLua doesn't touch in the `@rpath` and it have been working 
since forever for iOS https://github.com/NLua/KeraLua/blob/main/build/iOS/KeraLua.net6.0-maccatalyst.csproj#L41
KeraLua just use the `NativeRefernce` item from iOS SDK.
